### PR TITLE
#16 Create view for the homepage following wireframe

### PR DESCRIPTION
**Pull request by anikaSchwing**
screenshot from 2017-03-28 11-42-03

### CODE REVIEW

File: spec/controllers/pages_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe PagesController, type: :controller do
+
+  describe "GET #home" do
+    it "returns http success" do
+      get :home
+      expect(response).to have_http_status(:success)
```
Comment (arnoFleming) [inline spec/controllers/pages_controller_spec.rb]: maybe you also want to test if you have specific content:

beware: pseud-code
```ruby
expect(page).to have_content 'some text'
expect(page).not_to  have_link('A link', href: '/non-existant-href')
```

Comment (arnoFleming) [inline spec/controllers/pages_controller_spec.rb]: maybe move these tests to the view test, but I personaly hate view testing

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'pages/home'
```
Comment (leefreemanxyz) [inline config/routes.rb]: Arguably this should be set to the root path

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div class="fluid-container text-center cover">
+  <div class="cover-inner">
+    <img src="" />
```
Comment (leefreemanxyz) [inline app/views/pages/home.html.erb]: We'll possibly change this to a background image when we receive some more assets, but we can cross that bridge when we come to it.

File: app/views/pages/home.html.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<div class="fluid-container text-center cover">
```
Comment (leefreemanxyz) [inline app/views/pages/home.html.erb]: The bootstrap class is [container-fluid, not fluid-container](http://getbootstrap.com/css/#overview-container). (If you inspect your page in Chrome, you can see that fluid-container isn't applying any styles to it

**Pull request merged**